### ISSUE TITLE
Twenty Twenty-Two and Twenty-Three: user-created alt text does not over write auto-generated alt text on featured images

### ISSUE THREAD

**Issue by likethegoddess** [Automattic/themes#7304]
### Quick summary

Alt text added by a user to a featured image does not overwrite auto-generated alt text on the home/blog page. The user-added  alt text does replace the auto-generated text 

A larger discussion may be worth having about this auto-generated alt text. Right now, the alt text of featured images on the home/blog page is copied from the post title verbatim. For people using screen readers, this repeats the same information to the user, which is time-consuming and lacks added value. 

### Steps to reproduce

1) Go to a site using Twenty Twenty-Two or Twenty Twenty-Three. 
2) Add/edit a post with a featured image that includes user-entered alt text. 
3) Inspect code 

### What you expected to happen

When a user enters alt text for an image, it should overwrite the auto-generated alt text. 

### What actually happened

The auto-generated alt text remains. 

### Browser

Google Chrome/Chromium, Mozilla Firefox

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

None

**Comment by carolinan** [comment]
This is an accessibility feature, not a bug, if the "link to post" setting is enabled.
It is not mentioned if the link is enabled or not.
When the image is linked, and there is no publicly visible link text because the link only contains the image, the alt text becomes the link text. It then needs to describe the link, not the image itself.

_**Opinion:**_ The repetitive content is less of an issue than having a link with no or incorrect link text, and it is one of those things that 
a developer can easily fix in a PHP based theme because we can use conditional checks:
For example, the title and featured image can be inside one link, reducing the noise, but that can't be one with block themes at this point.

**Comment by carolinan** [comment]
(It is also not limited to these two themes since it is part of the block.)

**Comment by likethegoddess** [comment]
I know I've wrapped up a few concerns in the same issue and reasonable minds can disagree. What I consider a bug is the inability to replace the auto-generated alt text with my own alt text on featured images on the home/blog page.

**Comment by likethegoddess** [comment]
Thanks for your input, Caroline. I know I've wrapped up a few concerns in the same issue. What I am writing about as a bug here is the inability to replace the auto-generated alt text with my own alt text on featured images on the home/blog page. Sound like that needs to go under Gutenberg instead.
